src/components/Modal.jsx

```diff
@@ -0,0 +1,135 @@
+import "flowbite";
+import "./Modal.css";
+import CrossSvg from "./CrossSvg";
+import { useEffect, useState } from "react";
+import RenderIf from "./RenderIf";
+
+function FormElement(props) {
```

Comment:
We can also write it like this.
function FormElement({ value, index }) {
  const [state, setState] = useState(value[index]);